python/ray/tests/BUILD

```diff
@@ -295,6 +293,8 @@ py_test_module_list(
     "test_placement_group_mini_integration.py",
     "test_scheduling_2.py",
     "test_multi_node_3.py",
+    "test_multiprocessing.py",
+    "test_placement_group_2.py",
```

Comment:
the current PR will make `cluster.add_node` takes 1 second more than before, There are nearly 30 `cluster.add_node` in `test_placement_group_2.py`, it takes about 240s before this PR, So I think it is reasonable to increase the time limit of `test_placement_group_2` to avoid flaky tests.

Reply:
> cluster.add_node takes 1 second more than before

This actually sounds pretty bad... (I feel like it will increase the runtime of tests too long).

Why is this the case?